tests/e2e/test_writer_encoder.py

```diff
@@ -39,7 +42,10 @@ def test_writer_encoder(psycopg_cursor):
             SUM(_bigint),
             SUM(_float),
             STRING_AGG(_character_varying, ' ' ORDER BY _smallint),
-            MAX(_date)
+            STRING_AGG(_text, ' ' ORDER BY _smallint),
+            MAX(_date),
+            SUM((_json->>'value')::integer),
+            SUM((_jsonb->>'value')::integer)
```

Comment:
maybe it will be easier to just `SELECT *` and make an assetions with insered values:
```
rows = [
  (1, 2, 3, 1.23, "1", "-1", date(2019, 1, 1), '{"value": 1}', b'{"value": -1}'),
  (2, 2, 3, 1.23, "1", "-1", date(2019, 1, 1), '{"value": 1}', b'{"value": -1}'),
  ...
]
with WriterEncoder(psycopg_cursor, "public.e2e_test_e2e_test_1", schema) as writer_encoder:
  for row in rows:
    writer_encoder.append_tuple(row)

db_rows = [tuple(row) for row in psycopg_cursor.execute('SELECT * FROM public.e2e_test_e2e_test_1 ORDER BY _smallint')]
assert rows == db_rows
```